build-logic/convention/src/main/kotlin/AndroidApplicationFirebaseConventionPlugin.kt

```diff
@@ -41,15 +41,13 @@ class AndroidApplicationFirebaseConventionPlugin : Plugin<Project> {
                 "implementation"(libs.findLibrary("firebase.crashlytics").get())
             }
 
-            extensions.configure<ApplicationAndroidComponentsExtension> {
-                finalizeDsl {
```

Comment:
Hey @dturner, any idea why `finalizeDsl` was used here?

Reply:
It was the result of much hacking around with Firebase config. Happy to go with the simpler solution suggested by @SimonMarquis